packs/rackspace/actions/get_vm_ips.py

```diff
@@ -0,0 +1,44 @@
+from lib.action import PyraxBaseAction
+from lib.formatters import to_server_dict
+
+__all__ = [
+    'GetVMIPsAction'
+]
+
+class GetVMIPsAction(PyraxBaseAction):
+    def run(self, count=None, region=None, metadata=None):
+        if region:
+            cs = self.pyrax.connect_to_cloudservers(region=region)
+        else:
+            cs = self.pyrax.cloudservers
+
+        servers = cs.list()
+
+        result = []
+        for server in servers:
+            item = to_server_dict(server=server)
+
+            if metadata:
+                include = self._metadata_intersection(server=item,
+                                                      metadata=metadata)
+
+                if not include:
+                    continue
+
+            result.append(item['public_ips'][1])
```

Comment:
And since it's a list maybe we can just append and return all the ips - result.extend(public_ips)? :)